server.test.js

```diff
@@ -0,0 +1,48 @@
+const request = require('supertest');
+const server = require('./server');
+const db = require('./dbConfig');
+
+
+
+afterEach(async() => {
+    await db('games').truncate();
+})
+
+describe('Route handlers', () =>{
+    
+    describe('Post endpoint for games', () =>{
+        it('Should respond with a 422 status code if information is missing', async() =>{
+            const response = await request(server).post('/games')
+            .send({title: 'Super Smash Bros'});
+            expect(response.status).toBe(422);
+        });
+        it('Should respond witha 201 status code if object is successfully added', async() =>{
+            const response = await request(server).post('/games')
```

Comment:
Can add another test to the POST request tests making sure that the posted data is returned in the response